src/data-workspace/validation/validation-result-prop-types.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types'
+
+export const ValidationRuleViolationWithMetaDataPropTypes = PropTypes.shape({
+    metaData: PropTypes.shape({
+        displayDescription: PropTypes.string.isRequired,
+        displayInstruction: PropTypes.string.isRequired,
+        displayName: PropTypes.string.isRequired,
+        id: PropTypes.string.isRequired,
+        importance: PropTypes.string.isRequired,
+        leftSide: PropTypes.shape({
+            displayDescription: PropTypes.string.isRequired,
+        }).isRequired,
+        operator: PropTypes.string.isRequired,
+        rightSide: PropTypes.shape({
+            displayDescription: PropTypes.string.isRequired,
+        }).isRequired,
+    }),
+    validationRule: PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+    }),
+})
+
+export const ImportanceLevelPropTypes = PropTypes.oneOf([
+    'LOW',
+    'MEDIUM',
+    'HIGH',
+])
```

Comment:
Could (should?) be constants in a `constants.js` module as well

Reply:
do you mean the PropTypes or the array passed to it?

Reply:
I mean something like this:

```js
const LOW = 'LOW'
const MEDIUM = 'MEDIUM'
const HIGH = 'HIGH'
```

I just double checked and there are only two instances where the actual string literal is being specified:
1. `validation-config.js`
2. `validation-result-prop-types.js`

I think instead of creating constants for the string literal, you could re-use the `validationLevels` variable in `validation-result-prop-types.js`

Reply:
> I think instead of creating constants for the string literal, you could re-use the validationLevels variable in validation-result-prop-types.js

updated